src/components/ProjectMembersSummary/ProjectMemberSummary.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { Card } from '@material-ui/core';
+import React from 'react';
+import {
+  ProjectMembersSummary as ProjectMembersSummaryComponent,
```

Comment:
No need to alias if it isn't going to conflict with another name here 😉

Reply:
No longer has alias